Userland/Libraries/LibJS/AST.cpp

```diff
@@ -4811,4 +4811,44 @@ String const& SourceRange::filename() const
     return code->filename();
 }
 
+NonnullRefPtr<CallExpression> CallExpression::create(SourceRange source_range, NonnullRefPtr<Expression> callee, Vector<Argument> arguments)
```

Comment:
Is this supposed to copy the vector every time? It looks like yes per the previous code as it didn't move the augments in every case..? but the pattern of vector for create, span for constructor is not applied consistently to NewExpression either way.

Reply:
Uh, let's use `Span` more and copy less!